openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TestCaseRepository.java

```diff
@@ -673,6 +709,7 @@ public FeedRepository.TaskWorkflow getTaskWorkflow(FeedRepository.ThreadContext
     return super.getTaskWorkflow(threadContext);
   }
 
+  // TODO: Is this needed? cc @Harsha
   public static class TestCaseFailureResolutionTaskWorkflow extends FeedRepository.TaskWorkflow {
```

Comment:
@harshach since we are handling the task status when adding new status in the Incident Manager, not sure if these methods here are necessary anymore. Don't really know how these workflows come into play

Reply:
but the task flow happens when updating the Incident Manager status right? On my mind:
- When a test fails, we create a `NEW` incident and open a task to the test owner about "New Incident"
- The owner can `ACK` the incident
- The owner can move the Incident to `Assigned`. This will close the initial task and open a task for the assignee about "You should fix this"
- When the assignee/owner move the Incident to Resolved, the final task gets closed